core/build.vala

```diff
@@ -2,11 +2,11 @@ namespace Build {
     public const string APP_NAME = "noise";
```

Comment:
I don't really like hardcoding thinks like that. Meson offers a feature to write build info to a file: http://mesonbuild.com/Configuration.html